rpc/src/v1/impls/ibc.rs

```diff
@@ -57,56 +59,75 @@ where
         };
 
         let mut context = ibc::context::TopLevelContext::new(&mut state, block_number);
-        let mut client_manager = ibc::client::Manager::new(&mut context);
-        let client_state = client_manager.query(&client_id).map_err(|_| errors::ibc_client_not_exist())?;
-
-        let consensus_state = client_state.get_consensus_state(&mut context);
+        let client_manager = ibc::client::Manager::new(&mut context);
 
-        let rlp_encoded_consensus_state = consensus_state.encode();
+        let client_state = client_manager.query(&client_id).map_err(|_| errors::ibc_client_not_exist())?;
 
+        let result = client_state.rlp_bytes();
         Ok(Some(IBCQueryResult {
             block_number,
-            raw: rlp_encoded_consensus_state.to_hex(),
-            // FIXME
+            raw: result.to_hex(),
```

Comment:
It's okay to return the RLP encoded value here for now.
In the future, we should return the JSON formatted client state instead of RLP encoded value. Since the relayer should extract each field in the ClientState, returning JSON will be better.

Reply:
Ok